src/gui/gui.pri

```diff
@@ -19,7 +19,6 @@ HEADERS += \
     $$PWD/torrentcontenttreeview.h \
     $$PWD/deletionconfirmationdlg.h \
     $$PWD/statusbar.h \
-    $$PWD/ico.h \
```

Comment:
Probably. Git history shows that this was used almost since the inception of this program.
From the `ico.cpp` we used only one of the functions (`ICOHandler::canRead()`), the rest of the file was useless. (and that source doe was copied from kimgio)
Qt now(?) provides similar functionality, so the extra code was dropped.
PS: Both QIcon and ICOHandler::canRead(), fail on the piratebay favicon.